src/lib/util/Config.ts

```diff
@@ -115,17 +117,13 @@ export class Config {
 		return this.configContent.javaHome;
 	}
 
-	// FIXME: Not supported yet - the logic is not hooked up to the actual call
-	// Leaving this as-is instead of moving to getConfigValue() style
-	public isEngineEnabled(engine: ENGINE): boolean {
+	public async isEngineEnabled(engine: ENGINE): Promise<boolean> {
 		if (!this.configContent.engines) {
 			// Fast exit.  No definitions means all enabled.
 			return true;
```

Comment:
I left this, but we might want to remove it in 3.0, when would this ever occur?

Reply:
I don't think we need the assumption part since we now take the default values if Config doesn't have an entry.